website/content/docs/commands/events.mdx

```diff
@@ -0,0 +1,105 @@
+---
+layout: docs
+page_title: events - Command
+description: |-
+  The "events" command interacts with the Vault events subsystem.
+---
+
+# events
+
+The `events` command is used to subscribe to Vault
+[events](/vault/docs/concepts/events) via `events subscribe`. It shows a
```

Comment:
Follow-up question: what does "it shows a real-time display of events" mean? Is there a separate `events` subcommand that lists available events or do we just return a list of active events as part of a successful subscription call?

Reply:
There is no subcommand for showing available events. This returns an on-going list of all events that are happening, and will continue forever (as opposed to almost every other Vault command, which do one thing, return a response, and are done).